index.ts

```diff
@@ -64,21 +64,22 @@ async function createServer() {
             ]),
           },
         };
+        let isError: boolean = false;
```

Comment:
You don't have to declare a type. If you just put false, it is automatically infered. Inference is better than straight up decl